lib/RV/Admin/DateTimeFormat.php

```diff
@@ -90,6 +90,66 @@ public static function formatUTCTime($sDateTime)
         return $oTime->format("$time_format");
     }
 
+    /**
+     * A method that takes a date/time string and formats it using the user's
+     * locale information - i.e. using the appropriate date/time representation.
+     *
+     * @param string dateTime The date/time to be formatted, in a string format
+     * @return string The date/time in the user's locale format
+     */
+    public static function formatDateTime($sDateTime)
```

Comment:
Perhaps `formatLocalDateTime` to avoid confusion?

Reply:
Agreed, will update.